server/controllers 2/ContributionsController.js

```diff
@@ -0,0 +1,301 @@
+// Copyright 2020 Mehmet Baker
```

Comment:
The folders controllers 2, lib 2 and services 2; and the file server2.js must not be included in this commit. Did you copy them on purpose by the way?